mods/doors/init.lua

```diff
@@ -6,6 +6,17 @@ local _doors = {}
 _doors.registered_doors = {}
 _doors.registered_trapdoors = {}
 
+local function replace_old_owner_information(pos)
+	local meta = minetest.get_meta(pos)
+	local owner = meta:get_string("doors_owner")
+	if not owner or owner == "" then
```

Comment:
It would be easier to understand here if you formatted the code as below:

```lua
if owner and owner ~= "" then
    meta:set_string("owner", owner)
    meta:set_string("doors_owner", "")
end
```

Not only does it shorten the code by removing the need for the `return` and `else`, it also makes it more readably. Aside from that, :+1: